tests/Controllers/BeatmapsetsControllerTest.php

```diff
@@ -175,6 +176,55 @@ public function testBeatmapsetUpdateMetadataAsOwner($state)
         $this->assertSame($resultLanguageId, $beatmapset->language_id);
     }
 
+    public function testBeatmapsetUpdateOffsetDifferent()
```

Comment:
looking at the other tests, it looks like this can be just part of the other metadata tests (they're missing event count check so that can also be added)